tgui/packages/tgui/backend.js

```diff
@@ -7,8 +7,8 @@
  * the response with already existing state.
  */
 
-import { UI_DISABLED, UI_INTERACTIVE } from './constants';
 import { callByond } from './byond';
+import { UI_DISABLED, UI_INTERACTIVE } from './constants';
```

Comment:
Any reason you moved this around? Its not an issue, just curious

Reply:
Automatically moved by VSCode/extension